package.json

```diff
@@ -38,7 +38,7 @@
     "lint": "jscs . && jshint . --exclude-path .gitignore",
     "standalone": "browserify index.js --standalone Pjax > pjax.js",
     "build-debug": "browserify index.js --debug --standalone Pjax > pjax.js",
-    "tests": "tape -r ./tests/setup.js ./tests/**/*.js",
+    "tests": "tape -r ./tests/setup.js ./tests/lib/*.js ./tests/lib/proto/*.js",
```

Comment:
The `**` glob wasn't working (and it doesn't seem to be working properly on TravisCI either since only 40 tests were run instead of 77), so I changed this to be explicit.

Reply:
To elaborate, from what I understand, globs not enclosed in quotes might be parsed by bash, and then it's subject to the environment. When they are enclosed in quotes, like @MoOx said, they're parsed as CLI arguments and parsed by Node.